client/dashboard/data/index.ts

```diff
@@ -378,12 +378,9 @@ export const fetchEmail = ( id: string ): Promise< Email | undefined > => {
 };
 
 export const fetchUser = async (): Promise< User > => {
-	return await wpcom.me().get();
+	return wpcom.req.get( '/me' );
 };
 
 export const fetchTwoStep = async (): Promise< TwoStep > => {
-	return wpcom.req.get( {
-		path: '/me/two-step?http_envelope=1',
-		apiNamespace: 'rest/v1.1',
-	} );
```

Comment:
@youknowriad I'm simplifying this `/me/two-step` call: the extra decorations are not needed. `http_envelope` is added automatically, and the `apiVersion: '1.1'` is the default.

This is a common way how the library is misused in the codebase, I wonder where you copied this from 😁

Reply:
This is a question for @jorgefilipecosta I believe and I think he's AFK for some time :)

Reply:
My guess is that it was copied from the "modal" that appears when you go to "profile" that asks you to do reauth.